mmdeploy/codebase/mmrotate/models/roi_trans_roi_head.py

```diff
@@ -0,0 +1,97 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch
+
+from mmdeploy.core import FUNCTION_REWRITER
+
+
+@FUNCTION_REWRITER.register_rewriter(
+    'mmrotate.models.roi_heads.roi_trans_roi_head'
+    '.RoITransRoIHead.simple_test')
+def roi_trans_roi_head__simple_test(ctx, self, x, proposals, img_metas,
```

Comment:
We could add a Unit test for this rewriting in https://github.com/open-mmlab/mmdeploy/blob/master/tests/test_codebase/test_mmrotate/test_mmrotate_models.py

Reply:
Can I ask for help in adding a unit test?
I pushed the draft for the unit test.
But it throws the error like below;
```
        wrapped_model = WrapModel(roi_head, 'simple_test', img_metas=img_metas)
>       rewrite_outputs, is_backend_output = get_rewrite_outputs(
            wrapped_model=wrapped_model,
            model_inputs=model_inputs,
            deploy_cfg=deploy_cfg)

tests/test_codebase/test_mmrotate/test_mmrotate_models.py:457:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
mmdeploy/utils/test.py:591: in get_rewrite_outputs
    backend_outputs = get_backend_outputs(ir_file_path, model_inputs,
mmdeploy/utils/test.py:553: in get_backend_outputs
    backend_outputs = backend_model(backend_feats)
/opt/conda/lib/python3.8/site-packages/torch/nn/modules/module.py:1102: in _call_impl
    return forward_call(*input, **kwargs)
mmdeploy/backend/onnxruntime/wrapper.py:85: in forward
    self.io_binding.bind_input(
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <onnxruntime.capi.onnxruntime_inference_collection.IOBinding object at 0x7fb5a0ef9b20>, name = 'proposals_1', device_type = 'cpu', device_id = -1, element_type = dtype('float32'), shape = torch.Size([1, 1, 1])
buffer_ptr = 94579117147072

    def bind_input(self, name, device_type, device_id, element_type, shape, buffer_ptr):
        '''
        :param name: input name
        :param device_type: e.g. cpu, cuda
        :param device_id: device id, e.g. 0
        :param element_type: input element type
        :param shape: input shape
        :param buffer_ptr: memory pointer to input data
        '''
>       self._iobinding.bind_input(name,
                                   C.OrtDevice(get_ort_device_type(device_type), C.OrtDevice.default_memory(),
                                               device_id),
                                   element_type, shape, buffer_ptr)
E       RuntimeError: Error when binding input: Failed to find input name in the mapping: proposals_1

/opt/conda/lib/python3.8/site-packages/onnxruntime/capi/onnxruntime_inference_collection.py:381: RuntimeError
```

Maybe, the trouble part is feeding the input;
```
    model_inputs = {
        'x': x,
        'proposals': [proposals, labels],
    }

```

Reply:
does proposals have `labels` in it?